feature-libs/checkout/base/root/checkout-root.module.ts

```diff
@@ -40,7 +47,23 @@ export function defaultCheckoutComponentsConfig() {
 }
 
 @NgModule({
-  imports: [CheckoutEventModule],
+  imports: [
+    CheckoutEventModule,
+    RouterModule.forChild([
+      {
+        // @ts-ignore
+        path: null,
+        canActivate: [CmsPageGuard],
+        component: PageLayoutComponent,
+        data: {
+          cxRoute: 'orderConfirmation',
+          cxContext: {
+            [ORDER_ENTRIES_CONTEXT]: OrderConfirmationOrderEntriesContext,
```

Comment:
Just for my knowledge, I'm assuming this cxContext is used to differentiate a order type in order confirmation page, but can't seem to find how it's being used as it's note even being called 🤔 

If this works as how I think it should work, are we going to be dropping the replenishment order thank you component?

Reply:
> cxContext is used to differentiate a order type

yes it's used for import / export feature to "know" if it should place the "export to CSV" or "import from CSV" buttons on the page.

> but can't seem to find how it's being used

It is used, I answered it on Slack.

> if this works as how I think it should work, are we going to be dropping the replenishment order thank you component?

yep, that was my idea as well. added a todo for it.

Reply:
Will post on the DM. Miguel agrees in keeping the last 4 digits.

Reply:
to discuss